root_locals.tf

```diff
@@ -116,12 +116,12 @@ locals {
   judgment_export_bucket_read_access_roles = [local.tre_export_role_arn]
 
   // s3 internal bucket encryption
-  internal_s3_encryption_key_arn = local.environment == "intg" ? module.s3_internal_kms_key.kms_key_arn : ""
-  internal_bucket_key_enabled    = local.environment == "intg"
+  internal_s3_encryption_key_arn = contains(["intg", "staging"], local.environment) ? module.s3_internal_kms_key.kms_key_arn : ""
```

Comment:
Just a minor thing, could write the ternaries as: `internal_s3_encryption_key_arn = local.environment == "prod" ? "" : module.s3_internal_kms_key.kms_key_arn`

Feel free to ignore